monitor/grafana/zeebe.json

```diff
@@ -110,7 +110,7 @@
     {
       "collapsed": true,
       "datasource": {
-        "uid": "${DS_PROMETHEUS}"
+        "uid": "PBFA97CFB590B2093"
```

Comment:
I believe these changes are fine as it is auto generated when exporting the panel.